src/consumer/index.js

```diff
@@ -203,11 +204,12 @@ module.exports = ({
       })
 
       if (e.name === 'KafkaJSNumberOfRetriesExceeded' || e.retriable === true) {
-        logger.error(`Restarting the consumer in ${e.retryTime}ms`, {
+        const retryTime = e.retryTime || retry.initialRetryTime || initialRetryTime
```

Comment:
I think we need a growing `retryTime`, maybe we should embed this into RetriableError. We don't have to do this now, but this shouldn't be a fixed number.